### PR TITLE
Component/linechart

### PR DESCRIPTION
**Pull request by KingBosu**
Fixed the charts so they are functioning as intended

### CODE REVIEW

File: app/users/components/LineChart.tsx [hunk 1]
```diff
@@ -1,16 +1,26 @@
-import React from 'react'
+import React from 'react';
 import { Line } from 'react-chartjs-2';
+import {Chart as ChartJS} from 'chart.js/auto'
```
Comment (elidill653) [inline app/users/components/LineChart.tsx]: Try to be consistent with your semi colon usage

File: app/users/components/LineChart.tsx [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
 
+interface LineData {
+  day : string,
+  userProfits : number
+}
 
-
-const LineCharts = ({data}) => {
+const LineCharts = ({ chartData}) => {
+
```
Comment (elidill653) [inline app/users/components/LineChart.tsx]: Try to use a prettier in VS code to remove extra whitespace

File: app/users/page.tsx [hunk 3]
```diff
@@ -1,59 +1,34 @@
-'use client';
-import React from 'react';
+'use client'
```
Comment (elidill653) [inline app/users/page.tsx]: Lets see if we can render this in the backend by removing `use client`

File: app/users/page.tsx [hunk 4]
```diff
@@ -1,59 +1,34 @@
...
+import React, { useState } from 'react';
```
Comment (elidill653) [inline app/users/page.tsx]: I dont think you are using `useState` in this file so you can remove this import

File: app/users/profitData.tsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export const ProfitData = [
+  { day: "Monday", userProfits: 300 },
+  { day: "Tuesday", userProfits: 450 },
+  { day: "Wednesday", userProfits: 200 },
+  { day: "Thursday", userProfits: 600 },
+  { day: "Friday", userProfits: 350 },
+
+];
```
Comment (elidill653) [inline app/users/profitData.tsx:7]: Use prettier to format this file

**Review by elidill653 — CHANGES_REQUESTED** [review]
There are a few areas of improvement so update these then we can do another review

File: app/users/page.tsx (hunk 3 above)
Comment (KingBosu) [reply app/users/page.tsx]: Unable to render without due to useState from booking

Comment (KingBosu) [reply app/users/page.tsx]: The react-charts is also a client side render unfortunately this was part of the reason I could not get it to display before

**Review by elidill653 — APPROVED** [review]
Looks good

**Pull request merged**